packages/remix/src/ssr/utils.ts

```diff
@@ -123,6 +123,7 @@ export function getResponseClerkState(requestState: RequestState, context: AppLo
     __clerkJSVersion: getEnvVariable('CLERK_JS_VERSION', context),
     __telemetryDisabled: isTruthy(getEnvVariable('CLERK_TELEMETRY_DISABLED', context)),
     __telemetryDebug: isTruthy(getEnvVariable('CLERK_TELEMETRY_DEBUG', context)),
+    __environment: getEnvVariable('NODE_ENV', context) || getEnvVariable('ENVIRONMENT', context),
```

Comment:
For Remix we should recommend users to add the `ENVIRONMENT` env var in their configs.